public/API/API_GetUserSummary.php

```diff
@@ -103,7 +103,7 @@
 getAccountDetails($user, $userDetails);
 
 $retVal['ID'] = $userDetails['ID'];
-$retVal['Points'] = $userDetails['RAPoints'];
+$retVal['Points'] = $userDetails['RAPoints'] + $userDetails['RASoftcorePoints'];
```

Comment:
Same comment as above. Given the other fields that are being copied, this seems like it should just be the hardcore points. The breakdown will already be provided by the `getUserPageInfo` above (see also my comment on that).